packages/ui-components/src/components/range/range.scss

```diff
@@ -0,0 +1,93 @@
+@import '../../assets/styles/globals';
+
+:host {
+	/** 
+	 * @prop --range-width Width of the range slider
+	*/
+
+	--range-width: 600px;
+}
+
+.main {
+	width: var(--range-width);
+	margin: 24% auto;
+	position: relative;
+}
+
+.range-min-max {
+	@include kv-font-label-small-regular;
+	width: var(--range-width);
+	display: flex;
+	justify-content: space-between;
+	color: kv-color('neutral-5');
+	margin-top: $spacing-4x;
+}
+
+.slider {
+	width: var(--range-width);
+	height: 10px;
+	outline: none;
+	border-radius: $spacing;
+	background: kv-color('neutral-9');
+
+	&::-webkit-slider-thumb {
+		width: 48px;
+		height: 48px;
+		cursor: pointer;
+		z-index: 3;
+		position: relative;
+		/* stylelint-disable */
```

Comment:
Please add some justification or links to why you need to disable this. Just for future reference.

Reply:
@djcouto  i disabled here the lint because i was using -webkit-appearence: none and this property is deprecated. Now i replaced with appearence: none and I dont need to disable lint anymore so it will be removed.